client/dist/style.css

```diff
@@ -71,6 +71,5 @@
   height: 100px;
   margin: auto;
   background-color: white;
-  position: fixed;
```

Comment:
@the-trash-panda FYI - I updated this since the title seems to cover up the overview for certain products.